RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 18.7
 -----
+* [*] Editor: Removes snackbar notice when switching between HTML or Visual mode [https://github.com/wordpress-mobile/WordPress-Android/pull/15571]
```

Comment:
Thank you for adding a release note on this 🙇 
Adding this comment to push it on the next release!

Reply:
Moved this to 18.8 👍🏼